packages/plugins/Gfi/src/components/List.vue

```diff
@@ -0,0 +1,159 @@
+<!-- eslint-disable vue/no-v-html -->
+<template>
+  <v-scroll-x-reverse-transition>
+    <v-card class="plugin-gfi-list">
+      <v-card-title>
+        <v-icon>fa-list</v-icon>
+        {{ $t('common:plugins.gfi.list.header') }}
+      </v-card-title>
+      <v-card-subtitle v-if="maxPage > 1">
+        {{
+          `${$t('common:plugins.gfi.list.entry')} ${page * pageLength + 1} ${$t(
+            'common:plugins.gfi.list.to'
+          )} ${Math.min((page + 1) * pageLength, listFeatures.length)} ${$t(
+            'common:plugins.gfi.list.of'
+          )} ${listFeatures.length}`
+        }}
+        <v-pagination
+          v-model="_page"
+          color="primary"
+          :length="maxPage"
+          :current-page-aria-label="
+            $t('common:plugins.gfi.list.pagination.currentPage', {
+              page: _page,
+              maxPage,
+            })
+          "
+          :page-aria-label="
+            $t('common:plugins.gfi.list.pagination.page', {
+              page: _page,
+              maxPage,
+            })
+          "
+          :next-aria-label="$t('common:plugins.gfi.list.pagination.next')"
+          :previous-aria-label="
+            $t('common:plugins.gfi.list.pagination.previous')
+          "
+          :wrapper-aria-label="$t('common:plugins.gfi.list.pagination.wrapper')"
+        ></v-pagination>
+      </v-card-subtitle>
+      <v-list>
+        <v-list-item v-if="!visibleListFeatures.length">
+          {{ $t('common:plugins.gfi.list.emptyView') }}
+        </v-list-item>
+        <v-list-item
+          v-for="feature of visibleListFeatures"
+          :key="`gfi-feature-list-${feature.ol_uid}`"
+          :two-line="listText.length === 2"
+          :three-line="listText.length === 3"
+          @click="itemClick(feature)"
```

Comment:
```suggestion
          @click="setOlFeatureInformation(feature)"
```
Or would this be triggered immediately? If not, `itemClick` can be removed.

Reply:
https://github.com/Dataport/polar/pull/30/commits/a3d39c2b4700674af27be7ff1c514963a0b72af8